[16.0][MIG] ir_filters_multi_company: Migration to 16.0

TelmoSuarez opened the pull request.

MateuGForgeFlow committed:
[12.0][IMP] purchas_sale_inter_company: sync lines and moves

MiquelRForgeFlow committed:
[FIX] purchase_sale_inter_company: consider moves that contain same products

Imagine you have a customization that enables the split of a stock move into several moves (distributed in several pickings). Thus, we need to handle this case.

HekkiMelody committed:
[MIG] purchase_sale_inter_company: migrate sync pickings for v14

renda-dev committed:
[IMP] account_invoice_inter_company: Remove duplicated partner creation in tests

bealdav committed:
[ADD] 16.0 company_dependent_flag module

bealdav committed:
Apply suggestions from code review

Co-authored-by: Sylvain LE GAL <sylvain.legal@grap.coop>

Commit:
[UPD] Update company_dependent_flag.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #507 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update purchase_sale_inter_company.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-purchase_sale_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-purchase_sale_inter_company/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (40 of 40 strings)

Translation: multi-company-14.0/multi-company-14.0-purchase_sale_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-purchase_sale_inter_company/it/

OCA-git-bot committed:
Merge PR #461 into 14.0

Signed-off-by sebastienbeau

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #480 into 14.0

Signed-off-by sebastienbeau

OCA-git-bot committed:
[BOT] post-merge updates

sebastienbeau committed:
purchase_sale_inter_company: fix validation of several picking. self can have more than one record

renda-dev committed:
[REL] purchase_sale_inter_company: Generate new major version

OCA-git-bot committed:
Merge PR #524 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

bguillot committed:
[FIX] when action post an auto_generated invoice browse not made with sudo leads to access rights issues on origin invoice

renda-dev committed:
[FIX] account_invoice_inter_company: Remove self company

from eligible companies in `_find_company_from_invoice_partner`

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (7 of 7 strings)

Translation: multi-company-14.0/multi-company-14.0-base_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-base_multi_company/es_AR/

renda-dev committed:
[FIX] github pre-commit action: pin python version to 3.11 in repository

OCA-git-bot committed:
Merge PR #527 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
Merge PR #526 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[IMP] update dotfiles

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (12 of 12 strings)

Translation: multi-company-14.0/multi-company-14.0-product_supplierinfo_group_intercompany
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_supplierinfo_group_intercompany/it/

OCA-git-bot committed:
Merge PR #525 into 14.0

Signed-off-by sebastienbeau

OCA-git-bot committed:
[BOT] post-merge updates

deniscraciungabriel committed:
[14.0] ir_config_parameter_multi_company implementation

pedrobaeza committed:
[FIX] base_multi_company: Proper _check_company when multiple companies selected

Steps to reproduce:

- Create two companies C1 and C2.
- Create a product with C1 and C2 as allowed companies.
- Having C1 as main company, update quantity on hand to 100.
- Switch to C2 as main company, but check the C1 company to be seen as
  well.
- Now update quantity on hand to any other value for the entry of the
  company C1.

Result:

'Stock>Inventory adjustment' belongs to company 'C2' and 'Product'
(product_id: 'Product 1') belongs to another company.

Explanation:

This is because the computed field `company_id` that is used for
checking the product company vs the stock.move company (the error is not
accurate, as it says the location, but it's the move), is returning the
main company where the user is.

Solution:

Inject a context whenever we run `_check_company` for having available
the company involved, and use that company as value for the record being
checked if included in the allowed companies.

OCA-git-bot committed:
Merge PR #535 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update base_multi_company.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-base_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-base_multi_company/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (39 of 39 strings)

Translation: multi-company-14.0/multi-company-14.0-account_invoice_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_invoice_inter_company/es/

pedrobaeza committed:
[FIX] base_multi_company: context depend variable name

It was not changed after suggestion.

OCA-git-bot committed:
[UPD] addons table in README.md

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (23 of 23 strings)

Translation: multi-company-14.0/multi-company-14.0-product_supplierinfo_intercompany
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_supplierinfo_intercompany/es/

pedrobaeza committed:
[FIX] base_multi_company: Initialize variable in _check_company

For avodiding the crash if not entering any if.

OCA-git-bot committed:
[UPD] addons table in README.md

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (128 of 128 strings)

Translation: multi-company-14.0/multi-company-14.0-account_invoice_consolidated
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_invoice_consolidated/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (15 of 15 strings)

Translation: multi-company-14.0/multi-company-14.0-multicompany_configuration
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-multicompany_configuration/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (12 of 12 strings)

Translation: multi-company-14.0/multi-company-14.0-product_supplierinfo_group_intercompany
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_supplierinfo_group_intercompany/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (12 of 12 strings)

Translation: multi-company-14.0/multi-company-14.0-intercompany_shared_contact
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-intercompany_shared_contact/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (11 of 11 strings)

Translation: multi-company-14.0/multi-company-14.0-account_payment_other_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_payment_other_company/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (8 of 8 strings)

Translation: multi-company-14.0/multi-company-14.0-product_tax_multicompany_default
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_tax_multicompany_default/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: multi-company-14.0/multi-company-14.0-mail_template_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-mail_template_multi_company/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (10 of 10 strings)

Translation: multi-company-14.0/multi-company-14.0-product_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_multi_company/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: multi-company-14.0/multi-company-14.0-mail_multicompany
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-mail_multicompany/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (8 of 8 strings)

Translation: multi-company-14.0/multi-company-14.0-partner_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-partner_multi_company/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (8 of 8 strings)

Translation: multi-company-14.0/multi-company-14.0-res_company_code
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-res_company_code/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (40 of 40 strings)

Translation: multi-company-14.0/multi-company-14.0-purchase_sale_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-purchase_sale_inter_company/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (71 of 71 strings)

Translation: multi-company-14.0/multi-company-14.0-account_multicompany_easy_creation
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_multicompany_easy_creation/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (8 of 8 strings)

Translation: multi-company-14.0/multi-company-14.0-base_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-base_multi_company/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (9 of 9 strings)

Translation: multi-company-14.0/multi-company-14.0-account_invoice_inter_company_queued
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_invoice_inter_company_queued/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (9 of 9 strings)

Translation: multi-company-14.0/multi-company-14.0-product_category_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_category_inter_company/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: multi-company-14.0/multi-company-14.0-purchase_quick_intercompany
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-purchase_quick_intercompany/es/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (8 of 8 strings)

Translation: multi-company-14.0/multi-company-14.0-base_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-base_multi_company/it/

renda-dev committed:
[FIX] base_multi_company: apply '_name_search' fix to 'search_read'

pedrobaeza committed:
[FIX] base_multi_company: _check_company multi-record aware

Follow-up of #535

_check_company is not an `ensure_one` method, so we need to make
sure that if it's called with several records, and they don't
belong to the same company, everything is OK.

OCA-git-bot committed:
Merge PR #539 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

HekkiMelody committed:
[FIX]purchase_sale_inter_company: handle pickings with lots/serials

Previously an intercompany picking with tracked products would simply
throw an error.

With this fix, the method searches for a lot in the destination company
that matches the one in the source company (same name and same product).
A new lot is created by duplicating the original, if none is found.

HekkiMelody committed:
[FIX] purchase_sale_inter_company: sync moves with same product

This commit fixes a bug that occurred when the intercompany PO/SO and
therefore the pickings had multiple lines of the same product.

In that case, the quantities done would not be synced correctly,
but rather the larger quantity would be applied
to all destination moves. The receiving company would receive more
product than was ordered in the PO.

OCA-git-bot committed:
Merge PR #538 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #541 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update purchase_sale_inter_company.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-purchase_sale_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-purchase_sale_inter_company/

OCA-git-bot committed:
Merge PR #542 into 14.0

Signed-off-by aleuffre

OCA-git-bot committed:
[BOT] post-merge updates

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (40 of 40 strings)

Translation: multi-company-14.0/multi-company-14.0-purchase_sale_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-purchase_sale_inter_company/it/

OCA-git-bot committed:
Merge PR #523 into 14.0

Signed-off-by bguillot

OCA-git-bot committed:
[BOT] post-merge updates

matteoopenf committed:
[FIX]base_multi_company fix singleton   File "/opt/odoo/parts/OCA/multi-company/base_multi_company/models/base.py", line 21, in _check_company
    company_source_id = self.id
  File "/opt/odoo/odoo/odoo/fields.py", line 3825, in __get__
    raise ValueError("Expected singleton: %s" % record)
Exception

OCA-git-bot committed:
Merge PR #548 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

SirAionTech committed:
[FIX] base_multi_company: Check multiple companies

OCA-git-bot committed:
Merge PR #552 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

HekkiMelody committed:
[FIX] purchase_sale_inter_company: fix demo data

Just changing these fields to make it easier to test the module.

Technically a bug in "stock" module's demo data. Contact Jeff Lawson is
also the "address" of warehouse "Chicago 1".
The fields property_stock_customer and supplier are set to
"Inter-warehouse transit" on Jeff Lawson for company San Francisco
and not for company Chicago as it should be.

Since there are no routes set for location "Inter-warehouse transit",
creating an inter-company PO from Chicago to SF fails;
when the inter-company SO is automatically created, with delivery
address towards the Chicago Warehouse i.e. Jeff Lawson, the creation
of the SO's picking fails as there is no route available.

OCA-git-bot committed:
Merge PR #555 into 14.0

Signed-off-by aleuffre

chienandalu committed:
[FIX] purchase_sale_inter_company: sync confirmed unlocked purchase updates

When the purchase user updates the PO after the SO is generated and
always if both are confirmed and unlocked, we sync the new lines to update the demand.

TT46773

renda-dev committed:
[IMP] purchase_sale_inter_company: manage and sync state

OCA-git-bot committed:
Merge PR #560 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

francesco-ooops committed:
Added translation using Weblate (Italian)

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (8 of 8 strings)

Translation: multi-company-14.0/multi-company-14.0-stock_intercompany
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-stock_intercompany/it/

OCA-git-bot committed:
Merge PR #559 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update purchase_sale_inter_company.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-purchase_sale_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-purchase_sale_inter_company/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (3 of 3 strings)

Translation: multi-company-16.0/multi-company-16.0-company_dependent_flag
Translate-URL: https://translation.odoo-community.org/projects/multi-company-16-0/multi-company-16-0-company_dependent_flag/es/

eLBati committed:
REF ir_config_parameter_multi_company with company_id field

dora-jurcevic committed:
imp: icon position improuved on company dependent fields

OCA-git-bot committed:
[BOT] post-merge updates

sebastienbeau committed:
intercompany_shared_contact: fix wrong field computation and add test

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: multi-company-16.0/multi-company-16.0-company_dependent_flag
Translate-URL: https://translation.odoo-community.org/projects/multi-company-16-0/multi-company-16-0-company_dependent_flag/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (8 of 8 strings)

Translation: multi-company-14.0/multi-company-14.0-base_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-base_multi_company/it/

PicchiSeba committed:
[FIX] partner_multi_company: default value for res_partner.company_ids

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (39 of 39 strings)

Translation: multi-company-14.0/multi-company-14.0-account_invoice_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_invoice_inter_company/it/

OCA-git-bot committed:
Merge PR #575 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update partner_multi_company.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-partner_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-partner_multi_company/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (11 of 11 strings)

Translation: multi-company-14.0/multi-company-14.0-partner_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-partner_multi_company/it/

renda-dev committed:
[FIX] purchase_sale_inter_company: wrong field

OCA-git-bot committed:
Merge PR #583 into 14.0

Signed-off-by aleuffre

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (39 of 39 strings)

Translation: multi-company-14.0/multi-company-14.0-account_invoice_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_invoice_inter_company/it/

OCA-git-bot committed:
Merge PR #574 into 14.0

Signed-off-by legalsylvain

Commit:
[UPD] Update ir_config_parameter_multi_company.pot

OCA-git-bot committed:
[BOT] post-merge updates

ilyasProgrammer committed:
[FIX] product_multi_company: recompute variant companies

OCA-git-bot committed:
Merge PR #590 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update product_multi_company.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-product_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_multi_company/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (11 of 11 strings)

Translation: multi-company-14.0/multi-company-14.0-product_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_multi_company/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (45 of 45 strings)

Translation: multi-company-14.0/multi-company-14.0-purchase_sale_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-purchase_sale_inter_company/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: multi-company-14.0/multi-company-14.0-account_move_change_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_move_change_company/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (8 of 8 strings)

Translation: multi-company-14.0/multi-company-14.0-stock_intercompany
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-stock_intercompany/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (11 of 11 strings)

Translation: multi-company-14.0/multi-company-14.0-product_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_multi_company/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (11 of 11 strings)

Translation: multi-company-14.0/multi-company-14.0-partner_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-partner_multi_company/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: multi-company-14.0/multi-company-14.0-account_invoice_inter_company_sale
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_invoice_inter_company_sale/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: multi-company-14.0/multi-company-14.0-ir_config_parameter_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-ir_config_parameter_multi_company/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: multi-company-14.0/multi-company-14.0-product_supplierinfo_intercompany_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_supplierinfo_intercompany_multi_company/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: multi-company-14.0/multi-company-14.0-partner_contact_company_propagation
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-partner_contact_company_propagation/es/

renda-dev committed:
[FIX] product_multi_company: inherit from multi.company.abstract

toita86 committed:
[IMP] company_dependent_flag: black, isort, prettier

toita86 committed:
[MIG] company_dependent_flag: Migration to 14.0

OCA-git-bot committed:
Merge PR #593 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update product_multi_company.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-product_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_multi_company/

toita86 committed:
[IMP] company_dependent_flag: Improved the display of decoration class

renda-dev committed:
[IMP] purchase_sale_inter_company: remove user error on lines mismatch

renda-dev committed:
[IMP] base_multi_company: remove 'no_company_ids' field

As solved in https://github.com/odoo/odoo/pull/81344

OCA-git-bot committed:
Merge PR #599 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update base_multi_company.pot

Commit:
[UPD] Update partner_multi_company.pot

Commit:
[UPD] Update product_multi_company.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-base_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-base_multi_company/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-product_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_multi_company/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-partner_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-partner_multi_company/

OCA-git-bot committed:
Merge PR #596 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

toita86 committed:
[IMP]purchase_sale_inter_company: lock incoming picking validation if intercompany transfer

OCA-git-bot committed:
Merge PR #594 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update company_dependent_flag.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-company_dependent_flag
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-company_dependent_flag/

OCA-git-bot committed:
Merge PR #600 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update purchase_sale_inter_company.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-purchase_sale_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-purchase_sale_inter_company/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (48 of 48 strings)

Translation: multi-company-14.0/multi-company-14.0-purchase_sale_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-purchase_sale_inter_company/it/

OCA-git-bot committed:
[IMP] update dotfiles

OCA-git-bot committed:
Merge PR #570 into 14.0

Signed-off-by sebastienbeau

OCA-git-bot committed:
[BOT] post-merge updates

toita86 committed:
[FIX]company_dependent_flag: fixed missing label

Before this fix in a subview the label of a cpny dependent field was missing.
Now for each of these fields we check if we are in a form subview and we check if
they have a label if not it will be added.

renda-dev committed:
[ADD] partner_multi_company_pos

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (7 of 7 strings)

Translation: multi-company-14.0/multi-company-14.0-base_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-base_multi_company/pt_BR/

OCA-git-bot committed:
Merge PR #622 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update partner_multi_company_pos.pot

OCA-git-bot committed:
[BOT] post-merge updates

francesco-ooops committed:
Added translation using Weblate (Italian)

francesco-ooops committed:
Added translation using Weblate (Italian (it_IT))

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 75.0% (3 of 4 strings)

Translation: multi-company-14.0/multi-company-14.0-partner_multi_company_pos
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-partner_multi_company_pos/it/

OCA-git-bot committed:
Merge PR #621 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

MarinaAForgeFlow committed:
[IMP] account_invoice_inter_company: add setting to disable or enable intercompany invoicing

This commit adds a new setting called intercompany_invoicing that allows the user to activate or deactivate the intercompany invoicing. When positing an intercompany invoice, if one of the companies have the setting disabled, the other invoice will not be automatically created.

Commit:
[FIX] In case of dropshipping, partner_id is the end customer so ref_company_ids will be empty and the picking will not sync. Use sale_id.partner_id instead.

Commit:
[IMP] purchase_sale_inter_company: make locking of target invoices optional.

In some cases people still want to modify the invoice.

Commit:
[FIX] purchase_sale_inter_company: support 'no backorder'

When an incomplete sale order is delivered and "No backorder" is chosen, SO moves split in two while PO stays the same.
Aggregating writes per each PO move makes sure qty does not get overwritten.
Usually second qty is zero so the destination move becomes quantity zero and the picking was even cancelled.

OCA-git-bot committed:
Merge PR #637 into 14.0

Signed-off-by aleuffre

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #646 into 14.0

Signed-off-by aleuffre

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
Translated using Weblate (Spanish)

Currently translated at 93.7% (45 of 48 strings)

Translation: multi-company-14.0/multi-company-14.0-purchase_sale_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-purchase_sale_inter_company/es/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (8 of 8 strings)

Translation: multi-company-14.0/multi-company-14.0-product_tax_multicompany_default
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-product_tax_multicompany_default/it/

Commit:
[IMP] purchase_sale_inter_company: Activity instead of hard raise

A number of things can go wrong in trying to confirm the PO picking
when the SO picking is confirmed. Instead of raising an error and
thereby blocking the confirm of the SO picking, provide an option by
which the exception can be caught. The SO picking will still be
confirmed, but an activity will be posted for someone to deal with the
PO picking manually.

flotho reviewed (approved):
Hi,

Thanks,
easy code review LGTM

christian-ramos-tecnativa commented on ir_filters_multi_company/security/ir_filters.xml:
```diff
@@ -0,0 +1,12 @@
+        <field name="name">User-defined Filters multi company rule</field>
+        <field name="model_id" ref="base.model_ir_filters" />
+        <field name="domain_force">
+            ['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]
```
```suggestion
            ['|',('company_id','=',False),('company_id','in',company_ids)]
```
I think this is needed

OCA-git-bot committed:
Merge PR #656 into 14.0

Signed-off-by aleuffre

Commit:
[UPD] Update purchase_sale_inter_company.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-purchase_sale_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-purchase_sale_inter_company/

OCA-git-bot committed:
Merge PR #684 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update account_invoice_inter_company.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-account_invoice_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_invoice_inter_company/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (43 of 43 strings)

Translation: multi-company-14.0/multi-company-14.0-account_invoice_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_invoice_inter_company/it/

OCA-git-bot committed:
Merge PR #644 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update account_invoice_inter_company.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: multi-company-14.0/multi-company-14.0-account_invoice_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_invoice_inter_company/

Commit:
[FIX] purchase_sale_inter_company: fix no attribute 'company_id'

OCA-git-bot committed:
Merge PR #707 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

JordiMForgeFlow committed:
[IMP] purchase_sale_inter_company: return created SO

OCA-git-bot committed:
Merge PR #722 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

JordiMForgeFlow committed:
[IMP] account_invoice_inter_company: propagate invoice payment reference

OCA-git-bot committed:
Merge PR #728 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: multi-company-14.0/multi-company-14.0-partner_multi_company_pos
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-partner_multi_company_pos/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: multi-company-14.0/multi-company-14.0-partner_contact_company_propagation
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-partner_contact_company_propagation/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (47 of 47 strings)

Translation: multi-company-14.0/multi-company-14.0-account_invoice_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_invoice_inter_company/it/

OCA-git-bot committed:
[IMP] update dotfiles

OCA-git-bot committed:
Merge PR #485 into 14.0

Signed-off-by StefanRijnhart

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 20.0% (1 of 5 strings)

Translation: multi-company-14.0/multi-company-14.0-company_dependent_attribute
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-company_dependent_attribute/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (47 of 47 strings)

Translation: multi-company-14.0/multi-company-14.0-account_invoice_inter_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-account_invoice_inter_company/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: multi-company-14.0/multi-company-14.0-mail_multicompany
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-mail_multicompany/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: multi-company-14.0/multi-company-14.0-mail_template_multi_company
Translate-URL: https://translation.odoo-community.org/projects/multi-company-14-0/multi-company-14-0-mail_template_multi_company/it/
